### PR TITLE
serialization: GetRecord method identifier fix

### PR DESCRIPTION
**Pull request by krzysztof**
UUID was being serialized into the record identifier on the final XML instead of the pid_value, see: https://zenodo.org/oai2d?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai:zenodo.org:167082

### CODE REVIEW

**Comment by krzysztof** [comment]
@jirikuncar It looks like it was intended (and tested for in the unit test), however it's not how it was serialized in the legacy Zenodo.

File: invenio_oaiserver/response.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def getrecord(**kwargs):
 
     header(
         e_record,
-        identifier=str(pid.object_uuid),
+        identifier=str(pid.pid_value),
```
Comment (jirikuncar) [inline invenio_oaiserver/response.py]: IMHO it can be just `pid.pid_value` without `str(...)`.

Comment (krzysztof) [inline invenio_oaiserver/response.py]: Indeed :+1:

**Comment by lnielsen** [comment]
I think the build error can likely be fixed by just upgrading to ``sphinx>=1.5.1``

**Comment by jirikuncar** [comment]
@krzysztof there should be 2 commits - first one fixing the serialization and second one bumping the Sphinx version.

**Comment by krzysztof** [comment]
@jirikuncar This needs to be merged and requires a release: https://github.com/inveniosoftware/requirements-builder/pull/24

**Pull request merged**